conf/defaults.yaml

```
@@ -0,0 +1,116 @@
+##################### SigNoz Configuration Defaults #####################
+# Do not modify this file in signoz installs
+#----------------------------------------------
+# Version
+#----------------------------------------------
+version:
+  banner:
+    # Whether to show the banner on startup
+    enabled: true
+#----------------------------------------------
+# Instrumentation
+#----------------------------------------------
+instrumentation:
+  logs:
+    # Whether to enable log exports to an otlp compatible backend
+    enabled: false
+    # the level of the logger
+    level: error
+    processors:
+      - batch:
+          exporter:
+            otlp:
+              # the otlp endpoint
+              endpoint:
+              # the otlp protocol
+              protocol:
+  traces:
+    # Whether to enable trace exports to an otlp compatible backend
+    enabled: false
+    processors:
+      - batch:
+          exporter:
+            otlp:
+              # the otlp endpoint
+              endpoint:
+              # the otlp protocol
+              protocol:
+  metrics:
+    # Whether to enable metric exports to an otlp compatible backend or in prometheus format
+    enabled: true
+    readers:
+      - periodic:
+          exporter:
+            otlp:
+              # the otlp endpoint
+              endpoint:
+              # the otlp protocol
+              protocol:
+            prometheus:
+              host: 0.0.0.0
+              port: 9090
+#----------------------------------------------
+# Web
+#----------------------------------------------
+web:
+  # The prefix to serve web on
+  prefix: /
+  # The directory containing the static build files.
+  directory: /etc/signoz/web
+#----------------------------------------------
+# Cache
+#----------------------------------------------
+cache:
+  # Memory provider
+  provider: memory
+  ttl: -1
+  cleanup_interval: 60
+
+  # Options for redis
+  # provider: redis
+  # host: localhost
+  # port: 6379
+  # password: password
+  # db: 0
+#----------------------------------------------
+# Auth
+#----------------------------------------------
+jwt:
+  auth:
+    # The JWT secret for authentication
+    secret: secret
+#----------------------------------------------
+# Database
+#----------------------------------------------
+database:
+  # Supported options are sqlite
+  provider: sqlite
+  # Path to the sqlite database
+  path: /var/lib/signoz
+#----------------------------------------------
+# Storage
+#----------------------------------------------
+storage:
+  provider: clickhouse
+  dsn: tcp://localhost:9000
+  cluster: cluster
+  prometheus_config_path: /etc/signoz/config/prometheus.yml
+  max_idle_connections:
+  max_open_connections:
+  dial_timeout:
+  optimize_read_in_order_regex:
+  max_execution_time_leaf:
+  timeout_before_checking_execution_speed:
+  max_bytes_to_read:
```

Need better names here @srikanthccv!
```suggestion
storage:
  provider: clickhouse
  dsn: tcp://localhost:9000
  cluster: cluster
  prometheus_config_path: /etc/signoz/config/prometheus.yml
  max_idle_connections:
  max_open_connections:
  dial_timeout:
  optimize_read_in_order_regex:
  max_execution_time_leaf:
  timeout_before_checking_execution_speed:
  max_bytes_to_read:
```

---

prometheus:
   remote_read:
      config:
      url: